cmd/docker/daemon_none.go

```diff
@@ -7,5 +9,5 @@ import (
 // CmdDaemon reports on an error on windows, because there is no exec
 func (p DaemonProxy) CmdDaemon(args ...string) error {
 	return fmt.Errorf(
-		"`docker daemon` does not exist on windows. Please run `dockerd` directly")
+		"`docker daemon` does not exist on this platform. Please run `dockerd` directly")
```

Comment:
I've been meaning to submit a PR to change the Windows text. It should be is not supported on Windows. The none file is fine if you split the text between nine and Windows